src/routes/FavoritesScreen.tsx

```diff
@@ -0,0 +1,43 @@
+import { CommonActions, RouteProp, useFocusEffect } from '@react-navigation/native'
+import { StackNavigationProp } from '@react-navigation/stack'
+import React from 'react'
+
+import ServerResponseHandler from '../components/ServerResponseHandler'
+import VocabularyList from '../components/VocabularyList'
+import labels from '../constants/labels.json'
+import useLoadAsync from '../hooks/useLoadAsync'
+import { RoutesParams } from '../navigation/NavigationTypes'
+import AsyncStorage from '../services/AsyncStorage'
+
+interface FavoritesScreenProps {
+  route: RouteProp<RoutesParams, 'Favorites'>
+  navigation: StackNavigationProp<RoutesParams, 'Favorites'>
+}
+
+const FavoritesScreen = ({ navigation }: FavoritesScreenProps): JSX.Element => {
+  const { data, refresh, error } = useLoadAsync(AsyncStorage.getFavorites, {})
+
+  useFocusEffect(refresh)
+
+  return (
+    <ServerResponseHandler error={error} loading={false} refresh={refresh}>
+      {data && (
+        <VocabularyList
+          documents={data}
+          refreshFavorites={refresh}
+          onItemPress={(index: number) =>
+            navigation.navigate('VocabularyDetail', {
+              disciplineId: 0,
```

Comment:
@SteffiStuffel what did we decide here? There is obviously no discipline id, but it is used everywhere to save/read progress and so on...

Reply:
I think it is not used in VocabularyScreen

Reply:
It is for `VocabularyList` to save the progress.

Reply:
We have the VocabularyListScreen there it is used and needed. But in VocabularyDetailScreen it is not used and can be removed from route params.

Not sure if we should set it to `null` and check for non-null or if sticking to `0` is fine, seems to be less robust.

Reply:
I'll change it to `null` I guess